ietf/doc/views_ballot.py

```diff
@@ -185,9 +187,11 @@ def edit_position(request, name, ballot_id):
 
     balloter = login = request.user.person
 
-    if 'ballot_edit_return_point' in request.session:
-        return_to_url = request.session['ballot_edit_return_point']
-    else:
+    return_to_url = request.GET.get("ballot_edit_return_point")
+
+    if return_to_url is None:
+        print("*** DEFAULT return_to_url *********** ")
+        raise ValueError("ballot_edit_return_point is required param")
```

Comment:
This is a 'fail fast' approach to find incoming links that don't have the param. It will be removed later in dev.

Reply:
Removed